perf/lua/CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 set(TARANTOOL_BIN $<TARGET_FILE:tarantool>)
```

Comment:
probably a reason of `NO_DOC`, `NO_CHANGELOG`, `NO_TEST` should be `infrastracture.` Feel free to ignore.

Reply:
I've replaced with `perf test` as one that was used before.